apps/server/src/utils/parserFunctions.ts

```diff
@@ -142,14 +142,30 @@ export const parseViewSettings = (data: Partial<DatabaseModel>): ViewSettings =>
 /**
  * Sanitises an OSC Subscriptions array
  */
-export function sanitiseOscSubscriptions(subscriptions?: OscSubscription[]): OscSubscription[] {
+export function sanitiseOscSubscriptions(subscriptions: OscSubscription[], convert?: boolean): OscSubscription[] {
   if (!Array.isArray(subscriptions)) {
     return [];
   }
 
-  return subscriptions.filter(
-    ({ id, cycle, message, enabled }) =>
-      typeof id === 'string' && isOntimeCycle(cycle) && typeof message === 'string' && typeof enabled === 'boolean',
+  const sub = [...subscriptions];
+
+  if (convert) {
+    sub.forEach((value) => {
+      if (('message' in value) as unknown) {
+        value['address'] = value['message'];
+        delete value['message'];
+        value['payload'] = '';
+      }
+    });
+  }
+
+  return sub.filter(
+    ({ id, cycle, address, payload, enabled }) =>
+      typeof id === 'string' &&
+      isOntimeCycle(cycle) &&
+      typeof address === 'string' &&
+      typeof payload === 'string' &&
+      typeof enabled === 'boolean',
   );
 }
```

Comment:
@cpvalente is this the way to do it?

Reply:
I would personally not write the conversion, v3 is a major release and breaking changes are ok